pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2298,6 +2298,21 @@
             =/  rcvr  [ship her-life.channel.peer-core]
             "plea {<sndr rcvr bone=bone vane.plea path.plea>}"
         abet:(on-memo:peer-core bone plea %plea)
+      ::  +on-bane: handle request to delete a route
+      ::
+      ++  on-bane
+        |=  =ship
+        ^+  event-core
+        ?:  =(%czar (clan:title ship))
```

Comment:
I don't think we need the galaxy check.  We don't use galaxy lanes for anything iirc, but I don't see the benefit of adding logic to deny a request to delete one if it's there.

Reply:
From my local testing it appears that if set the `route` of a galaxy to `~` you'll never be able to talk to that galaxy again.